backend/src/v5/services/eventsListener/components/modelEvents.js

```diff
@@ -88,7 +88,7 @@ const revisionAdded = async ({ teamspace, project, model, revId, modelType, cali
 			{ _id: 0, tag: 1, author: 1, timestamp: 1, desc: 1, rFile: 1, format: 1, statusCode: 1, revCode: 1 });
```

Comment:
I think we need to allow the user to zoom out for more than the original image size.

For e.g. here, I'm looking at a corner of the picture and I can't zoom out to see the whole picture without re-centering the image, which doesn't feel natural:

![24-09-10 05-14-18](https://github.com/user-attachments/assets/ef9029d5-3a74-4163-96ea-c39d7375b67d)